### ISSUE TITLE
gluon.loss.SoftmaxCrossEntropyLoss() missed ignore_label parameter?

### ISSUE THREAD

**Issue by qiliux** [apache/incubator-mxnet#8404]
I found the `mx.symbol. SoftmaxOutput` has parameter ignore_label, however i can't find `gluon.loss.SoftmaxCrossEntropyLoss` has such parameter? 

if `gluon.loss.SoftmaxCrossEntropyLoss` doesn't have such parameter, how can i use loss function with ingore_label. https://github.com/apache/incubator-mxnet/pull/1938 pull have added `ignore_label` , how can i use it with `gluon`.

**Comment by chinakook** [comment]
use "sample_weight" instead

**Comment by kobenaxie** [comment]
Hi, could you give an example of using `ignore_label` in `gluon.loss.SoftmaxCrossEntropyLoss`  please ? thank you ~

**Comment by chinakook** [comment]
https://github.com/chinakook/U-Net
